pallets/providers/README.md

```diff
@@ -378,3 +377,45 @@ Error thrown when trying to get a root from a Main Storage Provider without pass
 ### `SpRegisteredButDataNotFound`
 
 Error thrown when a user has a Storage Provider ID assigned to it but its metadata data does not exist in storage (storage inconsistency error, should never happen).
+
+## Slashing Protocol
+
+Storage Providers who fail to submit a proof by the last challenge tick will be slashed, predetermined by the challenge period defined in the proofs-dealer pallet.
```

Comment:
"last challenge tick" might get confusing, I would change it to "Storage Providers who fail to submit a proof by their deadline" or something a bit more generic, considering this pallet should be agnostic on how Providers are marked as slashable.